### ISSUE TITLE
This code is google drive streaming right sir ?

### ISSUE THREAD

**Issue by yumedia** [p4v800m/google-drive-proxy-jwplayer#2]
Because now google drive devide into two ways when streaming, limit and no-limit view, if this code get google drive to download link so it will get limit view. 

So is this code limit view or not sir ?

**Comment by threathgfx** [comment]
it's a proxy system, so the usage will use directly from your server and no limitation will occur yet your bandwidth may affect.

**Comment by yumedia** [comment]
Is this code still working sir, I check but it not loading !

**Comment by p4v800m** [comment]
https://playerdb.cf/play/embed.php?id=dVN5M1o2ZGlHUXVrVjcyQ2o1M1VGR3JHUXlUKzQyU1VpU3p4UDYyOXh3UGVoVnlPVWk1NVl1blpxb2k2bnYvMFB5T2tsaEUxWGl1VlFWTG8yQ3FoZDF4cTdBVU1QbXhVSEZoMFFpMkF3SExnbkpMQUJlR1REVCtIZ3k2RTFxQmc=

**Comment by yumedia** [comment]
Yes, I tried, download all this code and upload to folder and try to run: domain.com/drive/embed.php?id=0B-LpD25Me2hCWW9ZcG1IY3Nycjg but it not loading, player just standing :(

**Comment by yumedia** [comment]
I think your script are using for playerdb.cf different with current script sp4v800m sir !

**Comment by yumedia** [comment]
In Readme.md you say:
Put google drive id on end of the url, just that.
https://domain.com/embed.php?id=0B9KF9ZZsSyo8YjBVTUg0c1d2d1U3

But your id link very long like: embed.php?id=dVN5M1o2ZGlHUXVrVjcyQ2o1M1VGR3JHUXlUKzQyU1VpU3p4UDYyOXh3UGVoVnlPVWk1NVl1blpxb2k2bnYvMFB5T2tsaEUxWGl1VlFWTG8yQ3FoZDF4cTdBVU1QbXhVSEZoMFFpMkF3SExnbkpMQUJlR1REVCtIZ3k2RTFxQmc=

**Comment by yumedia** [comment]
I also try with your api, not working also.

https://playerdb.cf/play/embed.php?id=0B9KF9ZZsSyo8YjBVTUg0c1d2d1U3

**Comment by p4v800m** [comment]
my api will not work the way it is doing because I use encryption at the beginning of the url and decrypted in the player, I modified my code to not display the url or google drive id but in the link above the example is functional, check it is doing something wrong the code is perfect. off the encryption and some modifications in the jwplayer that anyone can do is all the same.

**Comment by threathgfx** [comment]
@yumedia are you sure you installed it well? since you need to setup `servers.php`

**Comment by yumedia** [comment]
@p4v800m thank you sir, I'm waiting for you to update your github more exaplanations.
@threathgfx I put this code in my servers and use one domain only so I don't config anything in servers.php as explain in this line, anything wrong ?

if($domain =="") { 
$proxies[] = "drive.php";
} else {
$proxies[] = "".$domain."/drive.php";
}

**Comment by p4v800m** [comment]
give me domain.com/folder/embed.php link to check something

**Comment by thekksed** [comment]
So I tried your script on different servers resulted in different outcomes.

- On freehost 000webhost.com, the script is working.
http://dfdfrsef.000webhostapp.com/gdrive/embed.php?id=0B9KF9ZZsSyo8YjBVTUg0c1d2d1U3

- On hostinger (paid version of 000webhost), the script can play, but can't seek the video. Most likely 416 Requested Range Not Satisfiable.
http://mizii.ml/gdrive/embed.php?id=0B9KF9ZZsSyo8YjBVTUg0c1d2d1U3

- On a scaleway.com EU server, the script not working at all. It shows HTTP/1.1 403 Forbidden in Apache error log.
http://163.172.175.164/gdrive/embed.php?id=0B9KF9ZZsSyo8YjBVTUg0c1d2d1U3

**Comment by p4v800m** [comment]
freehost 000webhost.com - Working.
hostinger - Working.
scaleway.com - try to install LAMP with howtoforge.

all what has in common: very few hosts.
remember that this is a proxy server and a simple hosting will not account for so many accesses and transfers of videos with sizes and high qualities, in addition to the connection of these types of hosting are divided and very limited causing in small falls, try to use a dedicated or dependable servers like digitalocean, vultr, linode, aws and google. I have a player on google and it was not going to take any time to start. I have already used all the above servers and they are all horrible, mainly scaleway with several problems in OS. The player is functional and in excellent condition but for a performace the height needs to buy a service that gives the data correctly, 99% of the player needs an excellent server because the code downloads the videos on the server and sends to the user, if the quality is bad the player will also be.

**Comment by Zebraslive** [comment]
@p4v800m so it uses the same amount of bandwidth as if I where to host the video file myself ?

**Comment by yumedia** [comment]
@p4v800m I don't change anything in embed.php sir, all is default with your script !

**Comment by thekksed** [comment]
@p4v800m On hostinger it works, but we can't _seeking_ to another part of the video without redownloading the whole video again.  I wish we can find out the reason why it is getting error 416 Requested Range Not Satisfiable making the video not seekable. Because I'm also getting error 416 on a https://www.ovh.com/us/dedicated-servers/infra/173eg1.xml server.

**Comment by yumedia** [comment]
Yeah, this script still has some problem that can't play video @@

**Comment by yumedia** [comment]
One more question, does this scripts need ipv6 or turn off ipv6 to runnig sir? I'm using cloudflare !

**Comment by threathgfx** [comment]
What should I do to this error?
`curl_setopt($ch, CURLOPT_TCP_FASTOPEN, 1);`
![screenshot_6](https://user-images.githubusercontent.com/29234610/31802961-965ffd8a-b583-11e7-9bc5-bdcb5fcb690b.jpg)

**Comment by yumedia** [comment]
No support, code dead....

**Comment by Zebraslive** [comment]
@p4v800m does this script use the same amount of bandwidth as if I where to host the video on the server directly?

**Comment by yumedia** [comment]
No support !

**Comment by p4v800m** [comment]
Do you want to support yumedia? pay me, I'm not your official, I already offered a code that many would charge at least $ 50 and want support? Are you kidding? my escript works perfectly! visit filmesx.co and you will see!
